apis/core/index.ts

```diff
@@ -13,21 +18,21 @@ async function main() {
 
   app.enable('trust proxy')
 
-  app.get('/', (req, res) => {
-    return res.status(200).end()
-  })
-
   app.get('/env-config.js', uiConfig)
 
   app.use(await authentication())
-
-  app.get('/authenticated', (req, res) => {
-    return res.status(200).end()
+  await hydraBox(app, {
+    apiPath,
+    codePath,
+    baseUri,
+    sparql: {
+      endpointUrl: env.STORE_QUERY_ENDPOINT,
+      updateUrl: env.STORE_UPDATE_ENDPOINT,
+      storeUrl: env.STORE_GRAPH_ENDPOINT,
+    },
   })
 
-  app.get('/protected', guard().check('pipelines:read'), (req, res) => {
-    return res.status(200).end()
-  })
+  await bootstrap(env.STORE_GRAPH_ENDPOINT, baseUri)
```

Comment:
This is where the "static" resources from `bootstrap` directory are inserted into the store on startup